docs/servers/add-existing-server.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Add an existing Server
+intro: If you have servers provisioned manually or by other processes you can still connect them to your Devopness account and take the most of the Devopness infrastructure management features.
```

Comment:
Don't know the best way to put it on but i believe its important to have in the documentation that we don't support / recommend to add servers already used for any real operations on the user environment.

Reply:
@plantaz it seems to me that will belong to deeper level overview guides to be written in the future, whereas the current docs are simple steps to achieve a certain task, not being the steps responsible for educating the users on best practices. Does it make sense to you?

Reply:
Maybe we can change "If you have servers provisioned manually or by other processes" to "If you have servers provisioned outside of your desired environment at Devopness" because our users can add servers provisioned by Devopness on others environments as a self-hosted.

Reply:
Yes, makes sense! I just commented because in this case in particular we cannot guarantee that the server setup will not "break" any of their operation on this server. But maybe, as you suggested, will fit better in another place.